todo_app/app.py

```diff
@@ -1,22 +1,79 @@
 from flask import Flask, render_template, request
-from .data import session_items as sessionitems
 from todo_app.flask_config import Config
+import json
+import os
+import requests
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    items = sessionitems.get_items()
+    allCards = requests.get('https://api.trello.com/1/boards/602410634db0296b392dc99c/cards/open?key={}&token={}'.format(os.environ["TRELLO_KEY"], os.environ["TRELLO_TOKEN"] )).json()
```

Comment:
You can do this instead of templating the query string, though fine either way
```python
requests.get(
  'https://api.trello.com/1/boards/602410634db0296b392dc99c/cards/open',
  params={'key': os.environ["TRELLO_KEY"], 'token': os.environ["TRELLO_TOKEN"]}
)
```